skatepark/block-templates/search.html

```diff
@@ -12,7 +12,7 @@
 	<!-- wp:query {"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
 	<!-- wp:post-template -->
 
-		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"800"}},"fontSize":"huge","style":{"spacing":{"margin":{"top":"1em"}}}} /-->
+		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"var(--wp--custom--font-sizes--normal)","fontWeight":"800"}},"fontSize":"x-large","style":{"spacing":{"margin":{"top":"1em"}}}} /-->
```

Comment:
I think this is a dodgy find and replace

Reply:
yeah something like that :/